_data/sidebars/home_sidebar.yml

```diff
@@ -46,6 +46,8 @@
           url: /scheduler/mesosphere-dcos/kubernetes.html
         - title: Kubernetes (Other)
           url: /scheduler/kubernetes/install.html
+        - title: Kubernetes (Developer Edition)
```

Comment:
I don't we should add this to the sidebar, we can give the link to customers on a need-by-need basis.